pkg/cmd/tests/options.go

```diff
@@ -1,21 +1,390 @@
 package tests
 
 import (
+	"bytes"
+	"context"
+	"errors"
 	"fmt"
 	"os"
+	"os/exec"
 	"path"
 	"strings"
+	"sync"
+	"time"
 
 	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/formatter"
+	"github.com/onsi/ginkgo/v2/reporters"
+	"github.com/onsi/ginkgo/v2/types"
+	"github.com/onsi/gomega"
+	gomegaformat "github.com/onsi/gomega/format"
 	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/cmdutil"
 	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
 	"github.com/scylladb/scylla-operator/pkg/helpers/slices"
+	"github.com/scylladb/scylla-operator/pkg/signals"
 	ginkgotest "github.com/scylladb/scylla-operator/pkg/test/ginkgo"
+	"github.com/scylladb/scylla-operator/pkg/thirdparty/github.com/onsi/ginkgo/v2/exposedinternal/parallel_support"
+	"github.com/scylladb/scylla-operator/pkg/version"
 	"github.com/scylladb/scylla-operator/test/e2e/framework"
 	"github.com/spf13/cobra"
 	apierrors "k8s.io/apimachinery/pkg/util/errors"
+	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/klog/v2"
+	"k8s.io/kubectl/pkg/util/templates"
 )
 
+const (
+	parallelShardFlagKey            = "parallel-shard"
+	parallelServerAddressFlagKey    = "parallel-server-address"
+	randomSeedFlagKey               = "random-seed"
+	ginkgoOutputInterceptorModeNone = "none"
+)
+
+type RunOptions struct {
```

Comment:
> Move RunOptions to options.go where it belongs 

Putting aside the non-descriptiveness of such commit message, I think they should live where they are, near to the struct methods are in the same file as the struct definition. This is even more true when subcommands are used - these aren't generic options for *-test command but only for run subcommand. Maybe renaming the `options.go` to `testoptions.go` would avoid the confusion.

Consistency wise, all the other option live in the command files:
https://github.com/scylladb/scylla-operator/blob/7d9d80edb6a2a1ae76b2efe83a63712e9067fb24/pkg/cmd/operator/operator.go#L41
https://github.com/scylladb/scylla-operator/blob/5b752cca9f222188614191c4da1418955e915199/pkg/cmd/operator/gather.go#L42
https://github.com/scylladb/scylla-operator/blob/eefa9572f41540e29b48392882af21ca0e5633b9/pkg/cmd/operator/sidecar.go#L46

Anyway, I see no reason to couple such move to the scope of this PR. It will be also smaller and easier to review, with less friction points.

Reply:
Consistency wise, we don't have any as there're commands having options structure in the same file, as well as in separate ones (genapireference, releasenotes, tests), so it's a matter of personal preference.
Removed this improvement (IMO), to avoid this pointless discussion.

Reply:
>  to avoid this pointless discussion

This is not appropriate, nor on point. The bar (and the point) goes the other way. You chose to couple this opinionated change to the **existing code** to this PR (which didn't need it and could have had a diff order of magnitude smaller) so the *argument* really applies vice versa.